src/cluster_slot_stats.c

```diff
@@ -149,8 +190,11 @@ void clusterSlotStatsCommand(client *c) {
         int desc = 1, order_by = INVALID;
         if (!strcasecmp(c->argv[3]->ptr, "key-count")) {
             order_by = KEY_COUNT;
+        } else if (!strcasecmp(c->argv[3]->ptr, "network-bytes-in")) {
+            order_by = NETWORK_BYTES_IN;
         } else {
-            addReplyError(c, "Unrecognized sort metric for ORDER BY. The supported metrics are: key-count.");
+            addReplyError(c, "Unrecognized sort metric for ORDER BY. The supported "
+                             "metrics are: key-count and cpu-usec.");
```

Comment:
```suggestion
                             "metrics are: key-count and network-bytes-in.");
```

Reply:
I don't think we should mention the supported values in the error message, because then we will need to change the error message whenever we add more metrics.

The error messages don't need to be a help text.